segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/SystemJournal.java

```diff
@@ -1573,6 +1576,85 @@ private <R> CompletableFuture<R> executeExclusive(Callable<CompletableFuture<R>>
         }, this.executor);
     }
 
+    public CompletableFuture<Void> saveEpochInfo(int containerId, long containerEpoch, ChunkedSegmentStorage chunkedSegmentStorage) {
```

Comment:
Don't need to pass this parameter.
SystemJournal has reference chunkStorage already

Reply:
updated